src/sas/qtgui/Perspectives/Inversion/InversionPerspective.py

```diff
@@ -590,7 +632,8 @@ def removeData(self, data_list=None):
             data_list = [self._data]
         self.closeDMax()
         for data in data_list:
-            self._dataList.pop(data)
+            if data in self._dataList.keys():
```

Comment:
yes, we should check the key, but a quicker way to assure the op doesn't raise is to just pass `None`:

`self._dataList.pop(data, None)`

This will return the value if key exists in the dictionary, and None otherwise (no raise)